src/Android/Xamarin.Android/Samples/Map/OpenMapURL/OpenMapURL.cs

```diff
@@ -32,15 +32,15 @@ public class OpenMapURL : Activity
         // String array to hold urls to publicly available web maps
         private string[] itemURLs = new string[]
         {
-            "http://www.arcgis.com/home/item.html?id=2d6fa24b357d427f9c737774e7b0f977",
+            "http://www.arcgis.com/home/item.html?id=392451c381ad4109bf04f7bd442bc038",
             "http://www.arcgis.com/home/item.html?id=01f052c8995e4b9e889d73c3e210ebe3",
             "http://www.arcgis.com/home/item.html?id=92ad152b9da94dee89b9e387dfe21acd"
         };
 
         // String array to store titles for the webmaps specified above. These titles are in the same order as the urls above
         private string[] titles = new string[]
         {
-            "Housing with Mortgages",
+            "Population pressure",
```

Comment:
Nitpick: Everything else uses title case (including the webmap itself). Maybe capitalize Pressure to match?